From 8d518926a34883ecb69cc63ad4d37371f597a300 Mon Sep 17 00:00:00 2001
From: Dominic Hargreaves <dom@earth.li>
Date: Sun, 27 Nov 2011 16:15:42 +0000
Subject: Disable failing GNU/Hurd tests op/sigdispatch.t

These tests currently hang on GNU/Hurd, so disable them

Bug-Debian: http://bugs.debian.org/650188

Patch-Name: debian/hurd_test_skip_sigdispatch.diff
---
 t/op/sigdispatch.t |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/t/op/sigdispatch.t b/t/op/sigdispatch.t
index aa8a6c7..66e2a1e 100644
--- a/t/op/sigdispatch.t
+++ b/t/op/sigdispatch.t
@@ -9,7 +9,11 @@ BEGIN {
 use strict;
 use Config;
 
-plan tests => 17;
+if ($^O eq 'gnu') {
+    skip_all 'fails on GNU/Hurd (Debian #650188)' if $^O eq 'gnu';
+} else {
+    plan tests => 17;
+}
 
 watchdog(15);
 
